x-pack/test/security_solution_cypress/cypress/e2e/investigations/alerts/expandable_flyout/alert_details_right_panel.cy.ts

```diff
@@ -57,166 +59,160 @@ import { getNewRule } from '../../../../objects/rule';
-
-      // TODO figure out how to verify the toasts pops up
-      // cy.get(KIBANA_TOAST).should('be.visible').and('have.text', 'Successfully closed 1 alert.');
-      cy.get(EMPTY_ALERT_TABLE).should('exist');
-    });
-
-    // these actions are now grouped together as we're not really testing their functionality but just the existence of the option in the dropdown
-    it('should test other action within take action dropdown', () => {
-      expandFirstAlertExpandableFlyout();
-
-      cy.log('should add endpoint exception');
-
-      // TODO figure out why this option is disabled in Cypress but not running the app locally
-      //  https://github.com/elastic/security-team/issues/6300
-      openTakeActionButton();
-      cy.get(DOCUMENT_DETAILS_FLYOUT_FOOTER_ADD_ENDPOINT_EXCEPTION).should('be.disabled');
-
-      cy.log('should add rule exception');
-
-      // TODO this isn't fully testing the add rule exception yet
-      //  https://github.com/elastic/security-team/issues/6301
-      selectTakeActionItem(DOCUMENT_DETAILS_FLYOUT_FOOTER_ADD_RULE_EXCEPTION);
-      cy.get(DOCUMENT_DETAILS_FLYOUT_FOOTER_ADD_RULE_EXCEPTION_FLYOUT_HEADER).should('exist');
-      cy.get(DOCUMENT_DETAILS_FLYOUT_FOOTER_ADD_RULE_EXCEPTION_FLYOUT_CANCEL_BUTTON)
-        .should('be.visible')
-        .click();
-
-      // cy.log('should isolate host');
-
-      // TODO figure out why isolate host isn't showing up in the dropdown
-      //  https://github.com/elastic/security-team/issues/6302
-      // openTakeActionButton();
-      // cy.get(DOCUMENT_DETAILS_FLYOUT_FOOTER_ISOLATE_HOST).should('be.visible');
-
-      cy.log('should respond');
-
-      // TODO this will change when respond is improved
-      //  https://github.com/elastic/security-team/issues/6303
-      openTakeActionButton();
-      cy.get(DOCUMENT_DETAILS_FLYOUT_FOOTER_RESPOND).should('be.disabled');
-
-      cy.log('should investigate in timeline');
-
-      selectTakeActionItem(DOCUMENT_DETAILS_FLYOUT_FOOTER_INVESTIGATE_IN_TIMELINE);
-      cy.get(DOCUMENT_DETAILS_FLYOUT_FOOTER_INVESTIGATE_IN_TIMELINE_SECTION)
-        .first()
-        .within(() =>
-          cy.get(DOCUMENT_DETAILS_FLYOUT_FOOTER_INVESTIGATE_IN_TIMELINE_ENTRY).should('exist')
-        );
-    });
-  }
-);
+  // TODO this will change when add to new case is improved
+  //  https://github.com/elastic/security-team/issues/6298
+  it('should add to new case', () => {
+    expandFirstAlertExpandableFlyout();
+    openTakeActionButtonAndSelectItem(DOCUMENT_DETAILS_FLYOUT_FOOTER_ADD_TO_NEW_CASE);
+    fillOutFormToCreateNewCase();
+
+    cy.get(VIEW_CASE_TOASTER_LINK).should('contain.text', 'View case');
+  });
+
+  it('should mark as acknowledged', () => {
+    cy.get(ALERT_CHECKBOX).should('have.length', 1);
+
+    expandFirstAlertExpandableFlyout();
+    openTakeActionButtonAndSelectItem(DOCUMENT_DETAILS_FLYOUT_FOOTER_ADD_MARK_AS_ACKNOWLEDGED);
+
+    // TODO figure out how to verify the toasts pops up
```

Comment:
Instead of checking the toast, maybe you can navigate to acknowledged and verify there is 1 alert there? See `goToAcknowledgedAlerts` [here](https://github.com/elastic/security-team/issues/6303)

Reply:
is that link correct? I goes to an issue I created a while back, but not sure I see the relation with `goToAcknowledgedAlerts`.

I actually found the correct way to check the toast, so I think navigating to a different screen should not be necessary, as we have confirmation the action was successful!

Reply:
oops, I updated the [link](https://github.com/elastic/kibana/blob/main/x-pack/test/security_solution_cypress/cypress/e2e/investigations/alerts/changing_alert_status.cy.ts#L150). just a task that selects `acknowledged` filter

Reply:
that makes more sense, thank you!

Are you ok with just checking the toast and verifying the table has no longer any alerts? This avoids having to change page which should make the test run a tiny bit faster...